test/celeritas/ext/GeantImporter.test.cc

```diff
@@ -282,6 +282,14 @@ class OneSteelSphere : public GeantImporterTest
 class OneSteelSphereGG : public OneSteelSphere
 {
   protected:
+    void SetUp()
```

Comment:
```suggestion
    void SetUp() override
```

to make the CI happy!